snippets/primes.md

```diff
@@ -0,0 +1,14 @@
+### primes 
+
+It generates primes till a given number. 
+
+It works with the Sieve of Eratosthenes
```

Comment:
please explain about interbal workings of your function. how does it work? which built in functions you use to achieve your goal?